course/models.py

```diff
@@ -291,7 +291,14 @@ class Event(models.Model):
             # Translators: format of event kind in Event model
             help_text=_("Should be lower_case_with_underscores, no spaces "
             "allowed."),
-            verbose_name=_('Kind of event'))
+            verbose_name=_('Kind of event'),
+            validators=[
+                RegexValidator(
+                    "^" + EVENT_KIND_REGEX + "$",
+                    message=_("Should be lower_case_with_underscores, no spaces "
```

Comment:
@inducer Can you take a look at this? The validation error message is exactly the same with the help_text of the field. Will that look weird? Thanks.

Reply:
That's fine IMO.